frontend/src/components/commons/Notepad.jsx

```diff
@@ -103,57 +88,41 @@ class Notepad extends Component {
       <div style={styles.windowPadding}>
         {!notepadHidden && (
           <div style={styles.content}>
-            <div style={styles.hideNotepadBtnZone}>
-              <span
-                onClick={this.handleHide}
-                style={styles.hideNotepadBtnIcon}
-                className="fas fa-minus-circle"
-              />
+            <div style={styles.headerSection}>
+              <div style={styles.label}>
+                <label htmlFor={"text-area-notepad"}>
+                  {LOCALIZE.commons.notepad.title.toUpperCase()}
+                </label>
+              </div>
               <button onClick={this.handleHide} style={styles.hideNotepadBtn}>
+                <span style={styles.hideNotepadBtnIcon} className="fas fa-minus-circle" />
```

Comment:
Shouldn't it be an **<li>** tag instead of a **<span>** tag, since this an icon?